sympy/series/limits.py

```diff
@@ -109,6 +109,16 @@ def __new__(cls, e, z, z0, dir="+"):
         obj._args = (e, z, z0, dir)
         return obj
 
+
+    @property
+    def free_symbols(self):
+        e = self.args[0]
+        isyms = (e - self.args[1]).free_symbols
+        isyms.discard(self.args[1])
```

Comment:
No it's very fine :)  rather i mis-interpret it. I do not think that currently what you suggest is good though. Because
```
>>> x = Symbol('x', real=True)
>>> Limit(x + y, sin(x), 2).doit()    # here sin(x) can never approach `2`
x + y
>>> Limit(x + y, sin(x), 2).free_symbols
set([y])     # after applying what you suggest.
```
I also leading to wrong paths with this. With this we can not even know for sure `actually whether the answer(Limit.doit()) is really free of those returned symbols`

I rather find current commit to good though.